### ISSUE TITLE
"octo status" and "octo deploy" does not seem to work

### ISSUE THREAD

**Issue by Krishna-teja** [kneumei/hubot-octopus#5]
I tried testing both of those commands on Octopus deploy v3.2.2 and I couldn't get that working. I have tested out the octo promote and it works great.

**Comment by sheeley** [comment]
@kneumei: you might want to bump version and release to npm?

@Krishna-teja: for the interim, you can change your `package.json` to use the git repo as a dependency:

```
{
...
  "dependencies": {
    ...
    "hubot-octopus": "git@github.com:kneumei/hubot-octopus.git"
  }
}
```

**Comment by kneumei** [comment]
@Krishna-teja I just published hubot-octopus 0.1.0. I hope this resolves your issue. Sorry for the slow response, just haven't had time for this side project lately :(

Please let me know the status and deploy commands work for you now and I'll close this issue

**Comment by Krishna-teja** [comment]
@kneumei Thank you very much for the update. It works great!
